test/e2e/egress_test.go

```diff
@@ -311,6 +311,10 @@ func testEgressCRUD(t *testing.T, data *TestData) {
 				require.NoError(t, err, "Failed to check if IP exists on Node")
 				assert.False(t, exists, "Found stale IP on Node")
 			}
+			pool, err = data.crdClient.CrdV1alpha2().ExternalIPPools().Get(context.TODO(), pool.Name, metav1.GetOptions{})
+			assert.NoError(t, err)
+			assert.Equal(t, 1, pool.Status.Usage.Used, "ExternalIPPool status length not match")
```

Comment:
Maybe you can add `expectedTotal` to the table and check it too.

Reply:
Done